### DISCUSSION TITLE
scompaiono componenti aggiuntivi hassio

### DISCUSSION THREAD

**Discussion by maxdanluca** [indomus/forum#2414] [GENERALE - Discussioni generiche]
Spesso mi capita che tutti i componenti aggiuntivi smettano di funzionare. Cliccando su File editor, Node-red, Studio code server, Terminal appare una pagina totalmente bianca. Anche su impostazioni/componenti aggiuntivi non appare nulla. Per ripristinare devo spegnere e riaccendere il Raspberry perché Avvia non fa nulla. Come posso risolvere il problema?

**Comment by sky7176** [comment]
Ciao,
hai provato a guardare nei log se trovi info?
Su che sistema gira il tutto?
